packages/app/src/modules/dataTypes.js

```diff
@@ -49,19 +46,23 @@ export const dataTypes = {
     [EVENT_DATA_ITEMS]: {
         id: EVENT_DATA_ITEMS,
         name: i18n.t('Event data items'),
-        groupLabel: i18n.t('Select program'),
+        groupLabel: i18n.t('Program'),
         defaultGroup: null,
         groupDetail: false,
     },
     [PROGRAM_INDICATORS]: {
         id: PROGRAM_INDICATORS,
         name: i18n.t('Program indicators'),
-        groupLabel: i18n.t('Select program'),
+        groupLabel: i18n.t('Program'),
         defaultGroup: null,
         groupDetail: false,
     },
 };
 
+export const isProgram = (id, dataType) =>
```

Comment:
Instead of having this function, lets use the dataType configuration above, and add a property called "placeholder"